src/controllers/userController.js

```diff
@@ -32,21 +35,23 @@ const login = async (req, res) => {
     }
     const user = await User.findOne({ email });
 
-    const isPasswordCorrect = user && await user.comparePassword(password);
+    const isPasswordCorrect = user && (await user.comparePassword(password));
     if (!user || !isPasswordCorrect) {
       throw new UnauthenticatedError('Invalid Credentials');
     }
 
     const token = user.createJWT();
     const { email: userEmail, firstName, lastName } = user;
-    res.status(StatusCodes.OK).json({ user: { userEmail, firstName, lastName }, token });
+    res
+      .status(StatusCodes.OK)
+      .json({ user: { userEmail, firstName, lastName }, token });
   } catch (error) {
     console.error('Error logging in:', error);
     const statusCode = error.statusCode || StatusCodes.INTERNAL_SERVER_ERROR;
     const errorMessage = error.message || 'Error logging in';
     res.status(statusCode).json({ message: errorMessage });
   }
- };
+};
 
```

Comment:
I know this is unrealted to this pr, but lets make sure we invalidate the session cookie: 
const logout = (req, res) => {
  try {
    res
      .clearCookie('sessionToken', { httpOnly: true, secure: true, sameSite: 'strict' }) // Adjust options as needed
      .status(StatusCodes.OK)
      .json({ message: 'Logout successful' });
  } catch (error) {
    console.error('Error during logout:', error);
    res
      .status(StatusCodes.INTERNAL_SERVER_ERROR)
      .json({ message: 'Logout failed' });
  }
};

Reply:
Thank you Dan. We'll add it